release.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Exit script on error
+set -e
+
+# Ensure the version is passed as an argument
+if [ -z "$1" ]; then
+    echo "Usage: $0 <new_version>"
+    exit 1
+fi
+
+NEW_VERSION=$1
+
+# Validate the version follows semantic versioning (X.Y.Z)
+if [[ ! "$NEW_VERSION" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+    echo "Error: Version must be in format X.Y.Z (e.g., 1.10.4)"
+    exit 1
+fi
+
+# Pull the latest changes from the main branch
+echo "Pulling latest changes..."
+git pull origin main
```

Comment:
Future improvements to this script:

* Checkout to main
* Make sure you don't have a dirty git workspace or unstaged changes.

Reply:
Awesome! I made created a new linear task to capture this: https://linear.app/pyup/issue/SFTY-1234/further-release-automations